### PR TITLE
add-image-landingpage

### PR DESCRIPTION
**Pull request by sirtobiwan**
-image added
-image animation
-click on image to continue to next page

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+
+const StyledImage = styled.img`
+  width: 200px;
+  height: 200px;
+  border-radius: 10px;
+  object-fit: cover;
+  animation: ${zoomIn} 20s linear infinite;
+  overflow: hidden;
+  cursor: pointer;
+`;
```
Comment (djfarly) [inline pages/index.js:42]: Please use Image from next/image

```js
import Image from "next/image";
```

```suggestion
const StyledImage = styled(Image)`
  border-radius: 10px;
  object-fit: cover;
  animation: ${zoomIn} 20s linear infinite;
  overflow: hidden;
  cursor: pointer;
`;
```

File: pages/index.js [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
+    <IntroContainer>
+      <IntroParagraph>
+        How much money can you save with a solar panel?
+      </IntroParagraph>
+      <IntroSubtext>Click image below to find out!</IntroSubtext>
+      <StyledLink href="/radiation">
+        <StyledImage
+          src="https://images.unsplash.com/photo-1509391366360-2e959784a276?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1172&q=80"
+          alt="Solar Panel"
+        />
```
Comment (djfarly) [inline pages/index.js:15]: Add width and height here instead:
```suggestion
        <StyledImage
          src="https://images.unsplash.com/photo-1509391366360-2e959784a276?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1172&q=80"
          alt="Solar Panel"
          width="200"
          height="200"
        />
```

**Review by djfarly — CHANGES_REQUESTED** [review]
Hej Tobi,

thanks for the PR! See my note about using next/image below. Otherwise lgtm!

Keep going!

**Review by djfarly — CHANGES_REQUESTED** [review]
Hej Tobi,

thanks for the PR! See my note about using next/image below. Otherwise lgtm!

Keep going!

**Comment by sirtobiwan** [comment]
Thanks. I updated the next/image accordingly

**Pull request merged**